code/modules/mob/living/status_procs.dm

```diff
@@ -0,0 +1,522 @@
+// modify them
+
+// If you want a mob type to ignore a given status effect, just override the corresponding
+// `SetSTATE` proc - since all of the other procs are wrappers around that,
+// calling them will have no effect
+
+// BOOLEAN STATES
+
+/*
+	* EyesBlocked
+		Your eyes are covered somehow
+	* EarsBlocked
+		Your ears are covered somehow
+	* Resting
+		You are lying down of your own volition
+	* Flying
+		For some reason or another you can move while not touching the ground
+*/
+
+
+// STATUS EFFECTS
+// All of these decrement over time - at a rate of 1 per life cycle unless otherwise noted
+// Status effects sorted alphabetically:
+/*
+	*	Confused				*
+			Movement is scrambled
+	*	Dizzy						*
+			The screen goes all warped
+	*	Drowsy
+			You begin to yawn, and have a chance of incrementing "Paralysis"
+	*	Druggy					*
+			A trippy overlay appears.
+	*	Drunk						*
+			Essentially what your "BAC" is - the higher it is, the more alcohol you have in you
+	* EarDamage				*
+			Doesn't do much, but if it's 25+, you go deaf. Heals much slower than other statuses - 0.05 normally
+	*	EarDeaf					*
+			You cannot hear. Prevents EarDamage from healing naturally.
+	*	EyeBlind				*
+			You cannot see. Prevents EyeBlurry from healing naturally.
+	*	EyeBlurry				*
+			A hazy overlay appears on your screen.
+	*	Hallucination		*
+			Your character will imagine various effects happening to them, vividly.
+	*	Jitter					*
+			Your character will visibly twitch. Higher values amplify the effect.
+	* LoseBreath			*
+			Your character is unable to breathe.
+	*	Paralysis				*
+			Your character is knocked out.
+	* Silent					*
+			Your character is unable to speak.
+	*	Sleeping				*
+			Your character is asleep.
+	*	Slowed					*
+			Your character moves slower.
+	*	Slurring				*
+			Your character cannot enunciate clearly.
+	*	Stunned					*
+			Your character is unable to move, and drops stuff in their hands. They keep standing, though.
+	* Stuttering			*
+			Your character stutters parts of their messages.
+	*	Weakened				*
+			Your character collapses, but is still conscious.
+*/
+
+// DISABILITIES
+// These are more permanent than the above.
+// Disabilities sorted alphabetically
+/*
+	*	Blind	(32)
+			Can't see. EyeBlind does not heal when this is active.
+	*	Coughing	(4)
+			Cough occasionally, causing you to drop your items
+	*	Deaf	(128)
+			Can't hear. EarDeaf does not heal when this is active
+	*	Epilepsy	(2)
+			Occasionally go "Epileptic", causing you to become very twitchy, drop all items, and fall to the floor
+	*	Mute	(64)
+			Cannot talk.
+	*	Nearsighted	(1)
+			My glasses! I can't see without my glasses! (Nearsighted overlay when not wearing prescription eyewear)
+	*	Nervous	(16)
+			Occasionally begin to stutter.
+	*	Tourettes	(8)
+			SHIT (say bad words, and drop stuff occasionally)
+*/
+
+/mob/living
+
+	// Booleans
+	var/resting = FALSE
+
+	/*
+	STATUS EFFECTS
+	*/
+
+/mob // On `/mob` for now, to support legacy code
```

Comment:
I mean, anything that's not `/mob/living` isn't really a "real" mob that's part of the game world and its rules - they are all abstractions of some sort, being the `new_player`, the `observer`, and the `camera`, none of which should experience any of these states (I'm leaving out the spirit because that is ancient, bad code that's already on the chopping block)